presidio-analyzer/conf/analyzer.yaml

```diff
@@ -0,0 +1,52 @@
+recognizer_registry:
+  recognizers: 
+    - 
+      name: CreditCardRecognizer
+      supported_language: en
+
+    - 
+      name: CreditCardRecognizer
+      supported_language: es
+
+    - ItFiscalCodeRecognizer
+
+supported_languages: 
+  - en
+  - es
```

Comment:
Please break down to multiple like the credit card one

Reply:
hmmmm, are you sure? supported_languages is a list of strings that the engine receives, breaking it down will make it a list of "supported languages".

Reply:
You're right, my bad. I got confused